client/extensions/woocommerce/components/widget-group/index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import React, { Component, PropTypes } from 'react';
+
+/**
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/5512a8530a3394de9a0cedd0beb566762edddebc